notebooks/notebooks/iclr2022/win-rate-vs-training-steps.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import matplotlib.pyplot as plt
</pre><p>The code in this particular notebook feels fundamental enough that we likely want to move it <em>outside</em> of a notebook into a Python module -- but that can be another PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>